### ISSUE TITLE
Regeneration Syringe

### ISSUE THREAD

**Issue by CUBE0R** [studio-minus/ppg-bugs-and-requests#642]
**Feature description**
This is a syringe that increases natural regeneration by 2x, it can't heal things that natural regeneration cannot heal itself. Slows down acid degrading the body, making it take some time to take effect.

**Feature purpose**
Adds a new form of healing that isnt, instantly healing like a life syringe, or only minor healing specific issues like a mending syringe. 

**Additional context**
Would probably be a greenish or purple color.

**Comment by UniDuki** [comment]
2x faster is way too low, I suggest 10x or even 20x faster regeneration.
Maybe another idea that the regeneration could get faster the more liquid you put into the human.

**Comment by CUBE0R** [comment]
Good idea, i didnt realize just how slow regeneration was until I tested it.
